access/templates/access/_error_summary.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+
+<div class="error-summary">
+	<div class="error-summary__text">
+		<h4>{% trans "Submission failed" %}</h4>
+		<p>{% trans "Check your answers before trying again:" %}</p>
+		{% for field in result.form %}
+			{% if field.errors %}
+				<p>{{ field.label }}:
+				{% for error in field.errors %}
```

Comment:
If one field really has multiple errors, are the errors just concatenated inside one `<p>` here? Is there at least some space between them? Did you try it to see how it looks? Hmm, how could we set up multiple errors for testing?

Reply:
That is a good point. I only tested the outlook with multiple erroneous fields, but not if the same field has multiple errors. I checked it now by adding manually another error within the forms.py, as I couldn't think of away to introduce multiple errors at the same time with the current validations we have on those fields. I guess those cases luckily are quite rare. As if the format is wrong, only one error is raised saying to insert the correct format (eg. a number). And then again, if something is inserted, there is no error for the missing field.

Looks like there is always the dot and one space before the error. So the errors are listed automatically pretty nicely. :) I think this works as it is for those cases? 

![image](https://user-images.githubusercontent.com/43036333/90245929-411c5000-de3c-11ea-8e80-aa3c26a4776a.png)

Reply:
Yes, that is good enough :+1: